e2e-tests/functions

```diff
@@ -622,3 +622,31 @@ kubectl_bin() {
     rm "$LAST_OUT" "$LAST_ERR"
     return ${exit_status}
 }
+
+patch_secret() {
+    local secret=$1
+    local key=$2
+    local value=$3
+    
+    kubectl patch secret $secret -p="{\"data\":{\"$key\": \"$value\"}}"
+}
+
+getSecretData() {
+    local secretName=$1
+    local dataKey=$2
+    local  data=`kubectl get secrets/${secretName} --template={{.data.${dataKey}}} | base64 -D`
+    echo "$data"
+}
+
+check_mongo_auth() {
+    local command="$1"
+    local uri="$2"
```

Comment:
see comments on the results checks. This var became redundant as well. Anyway, its usage a bit confusing - why "find"?